script/Arcueid/Day 4/04_00_ARC04_3.txt

```diff
@@ -61,12 +66,13 @@ Hisui stares at me, then takes a small breath as if making up her mind before sh
 [sha:9752d4408e89ab93206a88574a8b769d5e0c4ddd]{
 -- Page 105, Offset 5822.
 -- そんな言葉だけでも、翡翠にとっては出過ぎた事だったのだろう。
-To say even those few words is very forward, coming from Hisui.
+Hisui appears to consider even these scant words as being too obtrusive.
+//WHAT. rewritten
```

Comment:
I understand wanting to respect the process, but the longer my comments, the more I have to write.
In this case I would rather just not make a comment at all if that's the case.
That's probably fine too.

Reply:
if you don't want to write a descriptive comment, no comment is a good alternative. What I want to avoid is little snippets of somebody's thought process being present and indiscernible in 3 months